File: spring/oauth2-client-rest-sample/src/main/java/com/github/yukihane/springboot/oauth2clientrestsample/MyController.java
package com.github.yukihane.springboot.oauth2clientrestsample;

import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
public class MyController {

    @GetMapping
    public String hello() {
        return "hello";
    }

}

File: spring/hello-thymeleaf-layout-dialect/src/main/kotlin/com/github/yukihane/examples/hello_thymeleaf_layout_dialect/MyController.kt
package com.github.yukihane.examples.hello_thymeleaf_layout_dialect

import org.springframework.stereotype.Controller
import org.springframework.web.bind.annotation.GetMapping

@Controller
class MyController {

    @GetMapping("/decorate/content1")
    fun content1(): String {
        return "decorate/content1"
    }

    @GetMapping("/decorate/content2")
    fun content2(): String {
        return "decorate/content2"
    }
}
